device-types/Cisco/C9200L-48PL-4X.yaml

```diff
@@ -0,0 +1,133 @@
+---
+manufacturer: Cisco
+model: Catalyst 9200L-48PL-4X
+part_number: C9200L-48PL-4X
+slug: cisco-c9200l-48pl-4x
+u_height: 1
+airflow: front-to-rear
+weight: 4.8
+weight_unit: kg
+is_full_depth: false
+comments: '[Cisco Catalyst 9200 Series Switches Data Sheet](https://www.cisco.com/c/en/us/products/collateral/switches/catalyst-9200-series-switches/nb-06-cat9200-ser-data-sheet-cte-en.html)'
+console-ports:
+  - name: Console
+    type: rj-45
+  - name: usb
+    type: usb-mini-b
+interfaces:
+  - name: GigabitEthernet1/0/1
+    type: 1000base-t
```

Comment:
Looking at the spec sheet some of these ports are POE+. POE+ ports need the following:
`poe_mode: pse`
`poe_type: type2-ieee802.3at`